### ISSUE TITLE
Example usage

### ISSUE THREAD

**Issue by jayvdb** [creativecuriositystudio/ng-comps#2]
Are there OSS projects which use this which can be examples for others to follow ?

**Comment by danielcavanagh** [comment]
Nah. This wasn't really built for externally use, and I can't imagine it provides anything worthwhile to you or anyone else

And, whilst I don't work at the company any more, it looks like it may have been abandoned

Maybe @thomasvogelaar can shed more light on it

**Comment by thomasvogelaar** [comment]
@danielcavanagh is right, this isn't used internally anymore. We're going to close this repository soon I think.

**Comment by jayvdb** [comment]
That is a shame.  It looks like the only useful/available frontend code for ModelSafe.
